sdk/textanalytics/Azure.AI.TextAnalytics/src/autorest.md

```diff
@@ -75,3 +75,20 @@ directive:
     $.properties.id["x-nullable"] = true;
     $.properties.text["x-nullable"] = true;
 ```
+
+### Remove BooleanResolution
+
+BooleanResolution is not supported in Text Analytics.
+
+``` yaml
+directive:
+- from: swagger-document
+  where: $.definitions
+  transform: >
+    delete $["BooleanResolution"]
+- from: swagger-document
+  where: $.definitions.BaseResolution.properties.resolutionKind.enum
+  transform: >
+    $.splice($.indexOf("BooleanResolution"), 1);
```

Comment:
Oh I see you mastered the JS dark magic
![MagicGIF](https://user-images.githubusercontent.com/6074665/222855353-9f1ef48b-f5e9-4f85-a813-57046297f94a.gif)

Reply:
I liked how clean and simple it was to do it this way. Magic indeed! 🪄✨